### PR TITLE
[Console] Added PHPDoc

### PR DESCRIPTION
**Pull request by atailouloute**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Component/Console/Helper/Table.php [hunk 1]
```diff
@@ -253,6 +274,13 @@ public function addRows(array $rows)
         return $this;
     }
 
+    /**
+     * Adds a row.
+     *
+     * @param $row
```
Comment (xabbuh) [inline src/Symfony/Component/Console/Helper/Table.php]: missing `string` type

File: src/Symfony/Component/Console/Helper/Table.php [hunk 2]
```diff
@@ -237,13 +244,27 @@ public function setHeaders(array $headers)
...
     {
         $this->rows = array();
 
         return $this->addRows($rows);
     }
 
+    /**
+     * Adds a list of rows.
+     *
+     * @param array $rows
```
Comment (xabbuh) [inline src/Symfony/Component/Console/Helper/Table.php]: should be `string[]`

File: src/Symfony/Component/Console/Helper/Table.php [hunk 3]
```diff
@@ -225,6 +225,13 @@ public function setColumnWidths(array $widths)
         return $this;
     }
 
+    /**
+     * Sets the headers.
+     *
+     * @param array $headers
```
Comment (xabbuh) [inline src/Symfony/Component/Console/Helper/Table.php]: `string[]`

Comment (atailouloute) [inline src/Symfony/Component/Console/Helper/Table.php]: Fixed

File: src/Symfony/Component/Console/Helper/Table.php (hunk 1 above)
Comment (atailouloute) [inline src/Symfony/Component/Console/Helper/Table.php]: It's actually a `string[]` or `TableSeparator`, I fixed that

File: src/Symfony/Component/Console/Helper/Table.php (hunk 2 above)
Comment (atailouloute) [inline src/Symfony/Component/Console/Helper/Table.php]: How to do so?

Comment (xabbuh) [inline src/Symfony/Component/Console/Helper/Table.php]: You can add it to the description.

Comment (atailouloute) [inline src/Symfony/Component/Console/Helper/Table.php]: Added, I used the same message used in addRow function 
`A row must be an array or a TableSeparator instance.`

File: src/Symfony/Component/Console/Helper/Table.php [hunk 4]
```diff
@@ -270,6 +298,14 @@ public function addRow($row)
...
 
+    /**
+     * Sets the row.
+     *
+     * @param int   $column
+     * @param array $row
+     *
+     * @return Table
+     */
     public function setRow($column, array $row)
```
Comment (fabpot) [inline src/Symfony/Component/Console/Helper/Table.php]: I would only keep the `@return` here. Everything else does not seem adding value.

File: src/Symfony/Component/Console/Helper/Table.php [hunk 5]
```diff
@@ -237,13 +244,27 @@ public function setHeaders(array $headers)
...
     {
         $this->rows = array();
 
         return $this->addRows($rows);
     }
 
+    /**
+     * Adds a list of rows, each row must must be an array or a TableSeparator instance.
+     *
+     * @param array $rows
```
Comment (fabpot) [inline src/Symfony/Component/Console/Helper/Table.php]: So, it should be `array|TableSeparator`?

File: src/Symfony/Component/Console/Helper/Table.php [hunk 6]
```diff
@@ -237,13 +244,27 @@ public function setHeaders(array $headers)
...
+     */
     public function setRows(array $rows)
     {
         $this->rows = array();
 
         return $this->addRows($rows);
     }
 
+    /**
+     * Adds a list of rows, each row must must be an array or a TableSeparator instance.
```
Comment (fabpot) [inline src/Symfony/Component/Console/Helper/Table.php]: The description of the param should be removed as it is/should be described in `@param`. The description should be removed altogether IMO as it does not add anything we cannot deduce from the method name.

**Comment by fabpot** [comment]
Closing as I don't think it adds so much value anyway.

**Pull request closed without merging**